### ISSUE TITLE
Cluster Bot Failing to start temp AWS cluster

### ISSUE THREAD

**Issue by astoycos** [openshift/ci-chat-bot#84]
sending `launch nightly` to the cluster bot

returns 

```
unable to find release image stream: imagestreams.image.openshift.io "release" is forbidden: User "system:serviceaccount:ci:ci-chat-bot" cannot get resource "imagestreams" in API group "image.openshift.io" in the namespace "ocp"
```

**Comment by astoycos** [comment]
Its back up but `launch 4.4` is failing with the following logs 

```
2020/05/12 18:23:15 ci-operator version v20200512-41d9bfd
2020/05/12 18:23:15 No source defined
time="2020-05-12T18:23:15Z" level=info msg="Parsed kubeconfig context: ci/api-ci-openshift-org:443"
2020/05/12 18:23:16 Ran for 0s
error: some steps failed:
  * could not resolve inputs: could not determine inputs for step [release:latest]: ImageStreamImport.image.openshift.io "release-import" is invalid: spec.images[0].from.name: Invalid value: "registry.svc.ci.openshift.org/\"ocp\"/release:4.4.3": invalid reference format
```

**Comment by smarterclayton** [comment]
Should be fixed now (i was able to run this)

**Comment by astoycos** [comment]
Looks like it's back up, thanks! @smarterclayton

**Comment by smarterclayton** [comment]
That was fixed in #86
